introduction.md

```diff
@@ -1,2 +1,4 @@
-Bow to your sensei.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?

Reply:
Good point! I'll do that right away..